### PR TITLE
Add component names to Props Table titles

### LINKED ISSUE #219: Update Site Page's PropsTable Titles
# Issue Description
PropsTable has a `componentName` prop which will add the component name to the PropsTable title. Index pages which display information on several components should be updated to use this prop. 

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior
Output like this:
screen shot 2017-10-26 at 5 06 13 pm

## Current Behavior
screen shot 2017-10-26 at 5 06 01 pm

### PR DESCRIPTION
**Pull request by Blackop778**
### Summary
Every documentation page now has the component's name in the `Props Table` title

### Additional Details
Closes #219 

Thanks for contributing to Terra. 
@cerner/terra

### CODE REVIEW

File: packages/terra-clinical-site/src/examples/action-header/Index.jsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const HeaderExamples = () => (
     <div id="version">Version: {version}</div>
     <Markdown id="readme" src={ReadMe} />
     <h2>Action Header</h2>
```
Comment (emilyrohrbough) [inline packages/terra-clinical-site/src/examples/action-header/Index.jsx]: lets remove `<h2>Action Header</h2>` as the title is now included in the props-table

File: packages/terra-clinical-site/src/examples/detail-view/Index.jsx [hunk 2]
```diff
@@ -20,9 +20,9 @@ const DetailViewExamples = () => (
     <div id="version">Version: {version}</div>
     <Markdown id="readme" src={ReadMe} />
     <h2>Detail View</h2>
```
Comment (emilyrohrbough) [inline packages/terra-clinical-site/src/examples/detail-view/Index.jsx]: lets remove `<h2>Detail View</h2>` as the title is now included in the appropriate props-table

File: packages/terra-clinical-site/src/examples/item-display/Index.jsx [hunk 3]
```diff
@@ -23,9 +23,9 @@ const ItemDisplayExamples = () => (
     <div id="version">Version: {version}</div>
     <Markdown id="readme" src={ReadMe} />
     <h2> Item Display </h2>
```
Comment (emilyrohrbough) [inline packages/terra-clinical-site/src/examples/item-display/Index.jsx]: lets remove `<h2>Item Display</h2>` as the title is now included in the appropriate props-table

File: packages/terra-clinical-site/src/examples/item-display/Index.jsx [hunk 4]
```diff
@@ -23,9 +23,9 @@ const ItemDisplayExamples = () => (
...
-    <PropsTable id="props" src={ItemDisplaySrc} />
+    <PropsTable id="props" src={ItemDisplaySrc} componentName="Item Display" />
     <h2> Comment Item Display </h2>
```
Comment (emilyrohrbough) [inline packages/terra-clinical-site/src/examples/item-display/Index.jsx]: lets remove `<h2>Comment Item Display</h2>` as the title is now included in the appropriate props-table

File: CONTRIBUTORS.md [hunk 5]
```diff
@@ -33,3 +34,4 @@ Cerner Corporation
 [@yusufali2205]: https://github.com/yusufali2205
 [@benbcai]: https://github.com/benbcai
 [@rakshith91]: https://github.com/rakshith91
+[@Blackop778]: https://github.com/Blackop778 s
```
Comment (StephenEsser) [inline CONTRIBUTORS.md]: Previewing this markdown file shows some formatting issues, it might be from this trailing s

**Pull request merged**